Disable compute batch intialization since it requires its own DRM context. This
is currently not supported by the GPU multiplexer and in turn corrupts the state
of the render batch.

diff --git a/src/lib/mesa/src/gallium/drivers/iris/iris_context.c b/src/lib/mesa/src/gallium/drivers/iris/iris_context.c
index 9fc6bb3..ac58e82 100644
--- a/src/lib/mesa/src/gallium/drivers/iris/iris_context.c
+++ b/src/lib/mesa/src/gallium/drivers/iris/iris_context.c
@@ -361,7 +361,8 @@ iris_create_context(struct pipe_screen *pscreen, void *priv, unsigned flags)
    }
 
    screen->vtbl.init_render_context(&ice->batches[IRIS_BATCH_RENDER]);
-   screen->vtbl.init_compute_context(&ice->batches[IRIS_BATCH_COMPUTE]);
+   //XXX: we need support multiple contexts in GPU driver
+   //screen->vtbl.init_compute_context(&ice->batches[IRIS_BATCH_COMPUTE]);
 
    return ctx;
 }
